core/src/data/value/json.rs

```diff
@@ -43,6 +43,7 @@ impl TryFrom<Value> for JsonValue {
             Value::I128(v) => JsonNumber::from_str(&v.to_string())
                 .map(JsonValue::Number)
                 .map_err(|_| ValueError::UnreachableJsonNumberParseFailure(v.to_string()).into()),
+            Value::F32(v) => Ok(JsonValue::Number(JsonNumber::from_f64(v as f64).unwrap())),
```

Comment:
1. We can use this
https://docs.rs/serde_json/latest/serde_json/value/enum.Value.html#impl-From%3Cf32%3E-for-Value

`Ok(v.into())` would be enough

2. unit test for `Value::F32` to `JsonValue::Number`